src/sinks/aws_cloudwatch_logs/config.rs

```diff
@@ -146,7 +146,7 @@ impl SinkConfig for CloudwatchLogsSinkConfig {
     }
 
     fn input(&self) -> Input {
-        Input::new(self.encoding.config().input_type())
+        Input::new(self.encoding.config().input_type() & DataType::Log)
```

Comment:
I opted to do this `&`ing rather than replacing the whole line with `Input::log()` to make it easier to relax as we review all of the sinks for data type support.